microsoft-edge/progressive-web-apps-chromium/webappmanifests.md

```diff
@@ -1,6 +1,6 @@
 ---
-description: Understand web app manifests in Progressive Web Apps
-title: Use web app manifests in Progressive Web Apps
+title: Deeper OS integrations via the Web App Manifest
+description: The Web App Manifest governs how your app looks and behaves within the context of the Operating System.
```

Comment:
Learn how to use the Web App Manifest to integrate your Progressive Web App into your Operating System.